### PR TITLE
Updates the Circuit Schematic Builder image and therefore the whole e…

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2920](https://openedx.atlassian.net/browse/DOC-2920)

@clrux made some updates to the Circuit Schematic Builder for AC-399, and I just got to updating the image. Of course, the problem it was based on had been updated in the meantime, so that meant updating the OLX...
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @clrux 
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check): @srpearce @pdesjardins  
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/exercises_tools/circuit_schematic_builder.rst [hunk 1]
```diff
@@ -12,7 +12,9 @@ grid. They then submit a DC, AC, or transient analysis of their circuit to the
 system for grading.
 
 .. image:: ../../../shared/images/CircuitSchematicExample.png
- :alt: Image of a circuit schematic builder.
+ :alt: A problem that asks learenres to use the circuit schematic builder to
```
Comment (srpearce) [inline en_us/shared/exercises_tools/circuit_schematic_builder.rst]: Typo: learners

File: en_us/shared/exercises_tools/circuit_schematic_builder.rst [hunk 2]
```diff
@@ -27,60 +29,129 @@ Create a Circuit Schematic Builder Problem
...
+   different times, so you'll earn a point only <i>after</i> you've performed
+   the transient simulation so that the checker will have a waveform to check!</p>
+   <solution>
+     <div class="detailed-solution"><p>Explanation:</p>
+     <p>The goal is to design a mixer circuit with characteristics of
+     \(V_{out}=\frac{1}{2}\cdot V_1+\frac{1}{6}\cdot V_2\)
+     You might have started to design your mixer with two resistors only as the example suggests.
+     But working through the math, soon you'll realize that the equations return no non-zero value for the resistor components.
+     Thus you have to change the design. The next simplest design will be to add a resistor \(R_3\) that connects the node Vout to ground. S
+     ee the schematic below:</p>
```
Comment (srpearce) [inline en_us/shared/exercises_tools/circuit_schematic_builder.rst]: Line wrap. Also, typo in line 114-115 (S / ee the schematic below).

File: en_us/shared/exercises_tools/circuit_schematic_builder.rst [hunk 3]
```diff
@@ -27,60 +29,129 @@ Create a Circuit Schematic Builder Problem
...
+     <img src="/static/images/circuits/lab2fmt.png"/>
+     <p>Since we are going to use only linear elements in this circuit (resistors are linear), superposition will hold
+     and thus one can look at the effect of each source \(V_1\) and \(V_2\) one at the time:</p>
```
Comment (srpearce) [inline en_us/shared/exercises_tools/circuit_schematic_builder.rst]: one at _a_ time

File: en_us/shared/exercises_tools/circuit_schematic_builder.rst [hunk 4]
```diff
@@ -27,60 +29,129 @@ Create a Circuit Schematic Builder Problem
...
+   worry if there is no solution, not if there are too many solutions. We will
+   simply find one of the many possible correct answers by arbitrarily
+   choosing a value for one of the variables later.</p>
+   <p>The first equation simplifies to \( R_1 = R_2\parallel R_3\) and the
+   second simplifies to \(R_2 = 5 \cdot R_1\parallel R_3\)
+   Expanding the notation gives: </p>
+   [mathjax]\frac{1}{R_1}=\frac{1}{R_2}+\frac{1}{R_3}
+   \tag{*} \\\frac{1}{R_1}+\frac{1}{R_3}=\frac{5}{R_2}  [/mathjax]
+ <p>Subtracting these two equations will yield \(R_2 = 2 \cdot R_3\)
+ And putting this back to the starred equation , will result in
```
Comment (srpearce) [inline en_us/shared/exercises_tools/circuit_schematic_builder.rst]: "back _into_ the starred equation"?

Extra space before comma (or we could just delete the comma)

File: en_us/shared/exercises_tools/circuit_schematic_builder.rst [hunk 5]
```diff
@@ -27,60 +29,129 @@ Create a Circuit Schematic Builder Problem
...
+ \(R_1 = \frac{2}{3} \cdot R_3\)
+ So now we have \(R_2\) and \(R_1\) in terms of \(R_3\) with the following ratios:</p>
+ [mathjax]R_2 = 2 \cdot R_3 \\ R_1 = \frac{2}{3} \cdot R_3 \\[/mathjax]
+ <p>Since the design hadn't mentioned anything about the resistances, one can
+ use a simple value of \(R_3= 3Ω\) and find the rest accordingly:</p>
+ [mathjax]R_1= 2Ω \\
+ R_2= 6Ω \\
+ R_3= 3Ω \\
+[/mathjax]
+ <p>With these resistor values, doing a transient analysis shows a result which meets the required specs of \(V_{out}\).</p>
```
Comment (srpearce) [inline en_us/shared/exercises_tools/circuit_schematic_builder.rst]: Line wrap

**Comment by srpearce** [comment]
A couple of typos—otherwise 👍

File: en_us/shared/exercises_tools/circuit_schematic_builder.rst (hunk 3 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/circuit_schematic_builder.rst]: I thought about correcting that, but it's straight from the problem text...

File: en_us/shared/exercises_tools/circuit_schematic_builder.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/circuit_schematic_builder.rst]: ditto

**Comment by downzer0** [comment]
The new image doesn't have the same stuff as the old one, but the buttons in the image (my work) look good 👍

**Comment by lamagnifica** [comment]
@clrux understood, the course team changed the problem in the interim!

**Pull request merged**